File: m3project/src/main/java/sg/edu/ntu/m3project/m3project/repository/SeatRepository.java
package sg.edu.ntu.m3project.m3project.repository;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import sg.edu.ntu.m3project.m3project.entity.SeatEntity;

@Repository
public interface SeatRepository extends CrudRepository<SeatEntity, String> {

}

File: m3project/src/main/java/sg/edu/ntu/m3project/m3project/repository/SeatRepository.kt
package sg.edu.ntu.m3project.m3project.repository;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import sg.edu.ntu.m3project.m3project.entity.SeatEntity;

@Repository
interface SeatRepository : CrudRepository<SeatEntity?, String?>
